File: SpringTILCode/src/main/java/com/example/springtilcode/global/jwt/UserAuthentication.java
package com.example.springtilcode.global.jwt;

import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
import org.springframework.security.core.GrantedAuthority;

import java.util.List;

public class UserAuthentication extends UsernamePasswordAuthenticationToken {
    public UserAuthentication(String principal, String credentials,
                              List<GrantedAuthority> authorities) {
        super(principal, credentials, authorities);
    }

}

File: kotlin/src/main/kotlin/com/example/demo/global/security/UserAuthentication.kt
package com.example.demo.global.security

import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
import org.springframework.security.core.GrantedAuthority

class UserAuthentication(principal: Any, credentials: Any?, authority: List<GrantedAuthority>?) :
    UsernamePasswordAuthenticationToken(principal, credentials, authority) {

}